vagrant/installer.sh

```diff
@@ -49,6 +49,19 @@ ansible-playbook -vvv -i ./inventory/hosts.ini configure-qat.yml | tee setup-qat
 
 # Kubernetes installation
 install_k8s
+if [[ "${CONTAINER_RUNTIME:-runc}" == "kata" ]]; then
```

Comment:
It does not have to be either ```runc``` or ```kata``` at this point. In kubernetes the pod can choose which runtime it needs to run via ```RuntimeClassName```.  Or is this switch to say "no kata this time"? 

Kata is not ```clear-linux-os``` specific either so we should check with regardless of the OS.

Reply:
You're right the only instruction that applies for CLR is the addition of the bundle, I'll fix it.

Reply:
Fixed on d86583c8d4e34dab3cc3c497870bc17b0a8e6481